Be sure we hit EOF while loading the configuration file on archs where char is
unsigned (ppc, arm)

Index: src/term.c
--- src/term.c.orig
+++ src/term.c
@@ -116,7 +116,7 @@ void term_loadall() {
    good_term *n;
    char line[MAXTERMCHARS+3];
    int i,j=0,l,t=0;
-   char c;
+   int c;
 
    fp = fopen(TERMSDIR "/" TERMSFILE,"r");
 
